app/viewmodels/helpers/messages/ViewMessageHelper.scala

```diff
@@ -258,14 +273,13 @@ class ViewMessageHelper @Inject()(
     }
   }
 
-  private[helpers] def contentForSubmittedVia3rdParty(isPortalSubmission: Boolean, relatedMessageType: String, ern: String, arc: String = "")
+  private[helpers] def contentForSubmittedVia3rdParty(draftMovementExists: Boolean, relatedMessageType: String, ern: String, arc: String = "")
                                                      (implicit messages: Messages): Seq[Html] = {
-    if (!isPortalSubmission) {
+    if (!draftMovementExists) {
       relatedMessageType match {
-        case "IE815" => Seq(p()(HtmlFormat.fill(Seq(
-          Html(messages("messages.submittedViaThirdParty.ie815")),
-          link(appConfig.emcsTfeCreateMovementUrl(ern), "messages.submittedViaThirdParty.ie815.link", id = Some("create-a-new-movement"), withFullStop = true)
-        ))))
+        //It's not possible to know if it was submitted via 3rd Party with EIS.
+        //This method will be removed as part of future alignment tickets. For now, only deal with IE815 and set this content to Empty
```

Comment:
Has this alignment been created?

Reply:
No but it's on Lexi's radar, I asked her on Friday about it.